src/main/java/com/google/devtools/build/lib/rules/genrule/GenRuleBaseRule.java

```diff
@@ -246,7 +246,7 @@ expanded to Windows style paths (with backslash).
 
         /* <!-- #BLAZE_RULE(genrule).ATTRIBUTE(output_to_bindir) -->
         <p>
-          If set to 1, this option causes output files to be written into the <code>bin</code>
+          If set to True, this option causes output files to be written into the <code>bin</code>
```

Comment:
Note that the documentation is rendered saying default is False and not default is 0
https://bazel.build/reference/be/general#genrule.local